lib/migration/config/v1/executor/container_vm.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor
+
+import (
+	"net/url"
+	"time"
+
+	"github.com/vmware/vic/pkg/version"
+)
+
+type State int
```

Comment:
Please add description here. I needed to dig deep to understand what this type is.